src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -106,11 +92,7 @@ internal DatabaseLocksManager LockedDatabaseManager
         /// Service host object for sending/receiving requests/events.
         /// Internal for testing purposes.
         /// </summary>
-        internal IProtocolEndpoint ServiceHost

```

Comment:
I think we make it a styling thing to put these on different lines @kburtram would know better than me

Reply:
I agree. Since these do not have get; set; bodies, I have seen single line usage and the terse usage is helpful reading code in my opinion.